tests/std/tests/P1222R4_flat_set/test.cpp

```diff
@@ -297,6 +335,15 @@ void test_erase_if() {
     assert(ranges::equal(fs, erased_result));
 }
 
+// TRANSITION, too simple
+void test_count() {
+    flat_set<int> fs{2};
+    assert(fs.count(1) == 0);
+
+    flat_multiset<int> fs2{1, 2, 2, 3};
+    assert(fs2.count(2) == 2);

```

Comment:
No change requested in this PR: It's not a huge deal, but in test code, I recommend "unique numbers for unique purposes". That is, if there's a `2` in the test code, ideally it should have the same meaning everywhere. Here, `2` is being used as an element value, and also as a number of elements. This test would be clearer if the elements were `10, 20, 20, 30`, for example.